src/lib/formats/jvc_dsk.cpp

```diff
@@ -170,6 +170,12 @@ bool jvc_format::parse_header(util::random_read &io, int &header_size, int &trac
 		break;
 	}
 
+	if (tracks>82)
+	{
+		osd_printf_verbose("jvc_format: track count of %d unsupported\n", tracks);
+		return false;
+	}
```

Comment:
Please put spaces around binary operators like the rest of the code in this file.

Reply:
Also it should probably use `osd_printf_info` like the other error messages.

Reply:
I switched to `info` and it appears during a failed probe. This is confusing:
`[MINGW64] C:\Users\Public\msys64\src\mame>mame coco3 -flop1 n80d.os9`
`jvc_format: track count of 160 unsupported`
I though JVC would be filtered out by the `.os9` file extension. But it doesn't appear to be so.